### PR TITLE
Execution detail page shows inputs and outputs

### PR DESCRIPTION
**Pull request by Bobgy**
Part of https://github.com/kubeflow/pipelines/issues/2086

video 1 - initial implementation: https://drive.google.com/file/d/1ZegAxrewO_fDLHOZUkJ7JixPmpoOeMkc/view
video 2 - deep link in artifact id: https://drive.google.com/file/d/1UFPrBf7-afdu80uOLvDZLynxEnkmKcMR/view
[Final screenshot](https://drive.google.com/file/d/1nAQ7xceCpgxznTGm2BKoTl2n9dXDahhJ/view?usp=sharing)
@neuromage Any feedback about this?

/area front-end
/kind feature
/assign @IronPan 
/cc @neuromage 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2155)

### CODE REVIEW

File: frontend/tsconfig.json [hunk 1]
```diff
@@ -1,10 +1,14 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```
Comment (Bobgy) [inline frontend/tsconfig.json]: Allows code like `import React, { Component } from 'react'`

**Comment by Bobgy** [comment]
Note, this is based on https://github.com/kubeflow/pipelines/pull/2153
Changeset will be more focused when it is merged.

**Comment by neuromage** [comment]
This is great, thanks @Bobgy!

Would it be possible to make the inputs/outputs linkable?

**Comment by Bobgy** [comment]
@neuromage it is already linkable. The first video were taken before I made the linkable feature.

**Comment by Bobgy** [comment]
Since @ajayalfred has LGTMed. I will merge this.
/approve

**Pull request merged**